### PR TITLE
[config/acl] Support everflow config translation

### PR DESCRIPTION
**Pull request by taoyl-ms**

### CODE REVIEW

**Comment by taoyl-ms** [comment]
Retest this please

**Comment by taoyl-ms** [comment]
Retest this please

File: src/sonic-config-engine/minigraph.py [hunk 1]
```diff
@@ -218,9 +218,10 @@ def parse_dpg(dpg, hname):
         aclintfs = child.find(str(QName(ns, "AclInterfaces")))
         acls = {}
         for aclintf in aclintfs.findall(str(QName(ns, "AclInterface"))):
-            aclname = aclintf.find(str(QName(ns, "InAcl"))).text
+            aclname = aclintf.find(str(QName(ns, "InAcl"))).text.lower().replace(" ", "_").replace("-", "_")
```
Comment (lguohan) [inline src/sonic-config-engine/minigraph.py]: what's the reason to change the aclname?

File: src/sonic-config-engine/minigraph.py [hunk 2]
```diff
@@ -229,9 +230,12 @@ def parse_dpg(dpg, hname):
                     print >> sys.stderr, "Warning: ACL " + aclname + " is attached to a Vlan interface, which is currently not supported"
                 elif port_alias_map.has_key(member):
                     acl_intfs.append(port_alias_map[member])
+                elif member.lower() == 'erspan':
```
Comment (lguohan) [inline src/sonic-config-engine/minigraph.py]: add comment here? If it is erspan, then attach the acl to all front panel ports.

File: src/sonic-config-engine/translate_acl [hunk 3]
```diff
@@ -127,29 +130,31 @@ def translate_acl_fixed_port(filename, output_path, port, max_priority):
...
-def translate_acl(filename, output_path, attach_to, max_priority):
+def translate_acl(filename, output_path, mini_acl, max_priority):
     yang_acl = pybindJSON.load(filename, openconfig_acl, "openconfig_acl")
     for aclsetname in yang_acl.acl.acl_sets.acl_set:
         tablename = aclsetname.replace(" ", "_").replace("-", "_")
-        if attach_to.has_key(tablename):
-            port = ','.join(attach_to[tablename])
+        if mini_acl.has_key(tablename):
+            is_mirror = mini_acl[tablename]['IsMirror']
+            port = ','.join(mini_acl[tablename]['AttachTo'])
```
Comment (lguohan) [inline src/sonic-config-engine/translate_acl]: port->ports

File: src/sonic-config-engine/minigraph.py (hunk 1 above)
Comment (taoyl-ms) [inline src/sonic-config-engine/minigraph.py]: To keep consistency with the name in acl.json

**Pull request merged**